### PR TITLE
Docs readme files

### PR DESCRIPTION
**Pull request by kirszenbaum**
Rewrote the following files:
* docs/README.md
* docs/README-EDITING.md
* docs/README-DEPLOYMENT.md

### CODE REVIEW

File: docs/README-DEPLOYMENT.md [hunk 1]
```diff
@@ -1,54 +1,61 @@
...
+    ```bash
+    docker login --registry docker.pkg.github.com
+    ```
+    * Login: Your GitHub account email
+    * Password: PAT with the `write:packages` permission: https://github.com/settings/tokens/new
 
-## From GH Action
+2. Deploy the docs:
+    ```bash
+    docker build -t docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:latest .
```
Comment (Siemienik) [inline docs/README-DEPLOYMENT.md:24]: npm run docs:docker:build
# npm run docs:docker:build:production # Production build

File: docs/README-EDITING.md [hunk 2]
```diff
@@ -1,62 +1,47 @@
...
+* To separate words, use `-`.
+* Use the `.md` file extension. The `.vue` extension is disabled.
 
-Since we want to support case-insensitive URLs, all links in JsDoc should:
+### Subdirectories
 
-* Before `#`: use `-` to separate words, lower case.
-* After `#`: lower case.
+* If possible, avoid adding new subdirectories.
+* When adding a new subdirectory, you have to update the sidebars logic.
```
Comment (Siemienik) [inline docs/README-EDITING.md:18]: These lines are outdated. Could you delete them?

File: docs/README-EDITING.md [hunk 3]
```diff
@@ -66,77 +51,120 @@ sitemap:
...
 
-### Live demo:
-It is able to run code snippets to show result of the code:
+A [`<semver.version>` directory](./README.md#handsontable-docs-directory-structure) with the largest version number gets tagged as the `latest` version of the docs.
 
-```
-    ::: example #example1
+To display the latest docs version in a browser:
+1. Start a [local Handsontable docs server](./README.md#getting-started-with-handsontable-docs).
+2. In your browser, go to http://localhost:8080/docs.
```
Comment (Siemienik) [inline docs/README-EDITING.md:74]: Please add information, that it is required to update both the latest and next version while editing the latest version.

File: docs/README-EDITING.md [hunk 4]
```diff
@@ -66,77 +51,120 @@ sitemap:
...
-**Release a version**
+    ```js
+        // code here
+    ```
 
-```shell script
-npm run docs:version <semver_number>
-# for instance 
-# npm run docs:version 9.0
+    :::
```
Comment (Siemienik) [inline docs/README-EDITING.md]: If could I ask you, please describe options for this md container

```
::: example #exampleId .class :preset --html 1 --js 2 --css 3  --no-edit --tab preview
```

* #example - container id 
* .class - container custom class 
* :preset - determine which dependencies should be loaded 

https://github.com/handsontable/handsontable/blob/c505c718ae9047a72ef9a970c7c2cf8d6b5369d8/docs/.vuepress/handsontable-manager/dependencies.js#L75-L91

* --html <pos>  --js <pos> --css <pos>  : determine positions of code snippets 

* --no-edit - do not add an `edit` button
* --tab - determine which tab is open as default, values:  (code|html|css|preview)

Comment (kirszenbaum) [reply docs/README-EDITING.md]: @Siemienik sure, check out the draft:

https://github.com/handsontable/handsontable/pull/8248/commits/74aff192e1bf94ce45990b19e6ff374e76b0f644

File: docs/README-DEPLOYMENT.md [hunk 5]
```diff
@@ -1,54 +1,63 @@
...
-```
+1. When deploying for the first time, log in to the GitHub Container Registry (ghcr.io):
+    ```bash
+    docker login --registry docker.pkg.github.com
+    ```
+    * Login: Your GitHub account email
+    * Password: PAT with the `write:packages` permission: https://github.com/settings/tokens/new
 
-## From GH Action
+2. Deploy the docs:
```
Comment (krzysztofspilka) [inline docs/README-DEPLOYMENT.md:22]: docs > documentation

File: docs/README-DEPLOYMENT.md [hunk 6]
```diff
@@ -1,54 +1,63 @@
...
+    ```bash
+    npm run docs:docker:build
+    # npm run docs:docker:build:production # Production build
 
-It happens automatically for each commit pushed into the `develop` branch.
+    docker push docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:latest
+    ```
 
-## Manually from GH Action
+### Deploying the docs from GitHub Actions
```
Comment (krzysztofspilka) [inline docs/README-DEPLOYMENT.md:30]: docs > documentation

File: docs/README-DEPLOYMENT.md [hunk 7]
```diff
@@ -1,54 +1,63 @@
...
 
-It is able to run manually deployment from any branch. To perform a deployment manually:
+GitHub Actions deploys the docs automatically for each commit pushed to the `develop` branch.
```
Comment (krzysztofspilka) [inline docs/README-DEPLOYMENT.md:32]: docs > documentation

Comment (krzysztofspilka) [reply docs/README-DEPLOYMENT.md:32]: for each commit > not sure about this, but maybe "after each commit" would be a better fit here?

File: docs/README-DEPLOYMENT.md [hunk 8]
```diff
@@ -1,54 +1,63 @@
...
 
-1. Go into `Actions` tab in a repo,
-2. On the left-hend side, select `documentation`,
-3. On the right-hand side, click `Run workflow`
-4. Select branch,
-5. Run workflow.
+### Manual deployment
 
-## Revert automatically deployment
+You can deploy the docs manually, from any branch:
```
Comment (krzysztofspilka) [inline docs/README-DEPLOYMENT.md:36]: docs > documentation

File: docs/README-DEPLOYMENT.md [hunk 9]
```diff
@@ -1,54 +1,63 @@
...
-docker pull docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:[COMMIT_HASH]
-docker tag docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:[COMMIT_HASH] docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:latest
-docker push docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:latest
-```
+* `:latest` - our server configuration watches for images with this tag.
+* `:[COMMIT_HASH]` - a backup.
+
+### Reverting a deployment
+
+To revert a docs deployment, run:
```
Comment (krzysztofspilka) [inline docs/README-DEPLOYMENT.md:51]: docs > documentation

File: docs/README-DEPLOYMENT.md [hunk 10]
```diff
@@ -1,54 +1,63 @@
...
 
-To revert deployment:
+GitHub Actions pushes the following tags to the GitHub Container Registry:
 
-```shell script
-docker pull docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:[COMMIT_HASH]
-docker tag docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:[COMMIT_HASH] docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:latest
-docker push docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:latest
-```
+* `:latest` - our server configuration watches for images with this tag.
```
Comment (krzysztofspilka) [inline docs/README-DEPLOYMENT.md:46]: our server configuration > the server configuration

File: docs/README-DEPLOYMENT.md [hunk 11]
```diff
@@ -1,54 +1,63 @@
...
+
+    ```bash
+    docker pull docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:[COMMIT_HASH]
+
+    docker tag docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:[COMMIT_HASH] docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:latest
+    
+    docker push docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:latest
+    ```
 
 ## Production environment:
```
Comment (krzysztofspilka) [inline docs/README-DEPLOYMENT.md:61]: Please remove this section - let's add it when it's ready.

File: docs/README-EDITING.md [hunk 12]
```diff
@@ -1,62 +1,42 @@
-# Handsontable Documentation Editing Guidelines
+# Docs editing guidelines
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:1]: docs > documentation

File: docs/README-EDITING.md [hunk 13]
```diff
@@ -1,62 +1,42 @@
...
 
-## Best practices for content
+This page covers guidelines for editing the [Handsontable docs](https://handsontable.com/docs).
 
-Applied to `/next` and `/*.*/`.
+## Maintenance rules
 
-### Mutating API Reference content
+When adding new docs files, check the docs [directory structure](./README.md#handsontable-docs-directory-structure), and follow the guidelines below.
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:7]: docs > documentation

File: docs/README-EDITING.md [hunk 14]
```diff
@@ -1,62 +1,42 @@
...
 
-* **title**: A header on a page.
-* **permalink**: Url to this page, **should be unique**.
-* **canonicalUrl**: Canonical url to the latest version of this page.
+### Sitemap front matter
 
-#### Sitemap headers
-
-The `vuepress-plugin-sitemap` give us additional headers to customise it behaviour:
+The [`vuepress-plugin-sitemap`](https://www.npmjs.com/package/vuepress-plugin-sitemap) lets us use additional tags to customize the docs site map:
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:39]: docs > documentation

File: docs/README-EDITING.md [hunk 15]
```diff
@@ -1,62 +1,42 @@
...
 canonicalUrl: /api/
 ---
-
-# Introduction
 ```
 
-* **title**: A header on a page.
-* **permalink**: Url to this page, **should be unique**.
-* **canonicalUrl**: Canonical url to the latest version of this page.
+### Sitemap front matter
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:37]: front matter > Frontmatter (https://vuepress.vuejs.org/guide/frontmatter.html)

File: docs/README-EDITING.md [hunk 16]
```diff
@@ -66,77 +46,130 @@ sitemap:
 ---
 ```
 
-### Subdirectories
+## Editing the docs
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:49]: docs > documentation

File: docs/README-EDITING.md [hunk 17]
```diff
@@ -66,77 +46,130 @@ sitemap:
...
 
-I recommend to keep files flat without subdirectories.
-If subdirectory being to add, it is required to update sidebars creating logic.
+When editing the docs content, follow the guidelines below.
 
-### Prefer .md, avoiding .vue
+Files to be included in the docs' next version live in the `next` directory.
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:53]: docs > documentation

Comment (krzysztofspilka) [reply docs/README-EDITING.md:53]: live > are kept/stored/hosted/placed/located?

File: docs/README-EDITING.md [hunk 18]
```diff
@@ -66,77 +46,130 @@ sitemap:
...
 
-The `*.vue` allows us to use vue pages and components. It is undesirable because it introduces additional complexity.
-For that purposes `.vue` suffix is disabled.
+The `next` version of the docs is available only locally and on the staging server.
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:55]: docs > documentation

File: docs/README-EDITING.md [hunk 19]
```diff
@@ -66,77 +46,130 @@ sitemap:
...
 
-Keeping all content as just `.md` file makes them more manageable.
+To display the `next` version in a browser:
+1. Start a [local Handsontable docs server](./README.md#getting-started-with-handsontable-docs).
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:58]: docs > documentation

File: docs/README-EDITING.md [hunk 20]
```diff
@@ -66,77 +46,130 @@ sitemap:
...
+2. In your browser, go to http://localhost:8080/docs/next/.
 
-## Editing documentation content
+### Editing the `latest` docs version
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:61]: docs > documentation

File: docs/README-EDITING.md [hunk 21]
```diff
@@ -66,77 +46,130 @@ sitemap:
...
 
-### Live demo:
-It is able to run code snippets to show result of the code:
+A [`<semver.version>` directory](./README.md#handsontable-docs-directory-structure) with the largest version number gets tagged as the `latest` version of the docs.
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:63]: docs > documentation

File: docs/README-EDITING.md [hunk 22]
```diff
@@ -66,77 +46,130 @@ sitemap:
...
 
-```
-    ::: example #example1
+If you're editing the `latest` version, remember to make the same edits to the `next` version as well.
 
-    ```js
-        // code here
-    ```
+To display the latest docs version in a browser:
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:67]: docs > documentation

File: docs/README-EDITING.md [hunk 23]
```diff
@@ -66,77 +46,130 @@ sitemap:
...
+1. Start a [local Handsontable docs server](./README.md#getting-started-with-handsontable-docs).
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:68]: docs > documentation

File: docs/README-EDITING.md [hunk 24]
```diff
@@ -66,77 +46,130 @@ sitemap:
...
+2. In your browser, go to http://localhost:8080/docs.
+
+### Editing an archived docs version
+
+To edit an archived docs version, go to the required [`<semver.version>` directory](./README.md#handsontable-docs-directory-structure).
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:73]: docs > documentation

File: docs/README-EDITING.md [hunk 25]
```diff
@@ -66,77 +46,130 @@ sitemap:
...
 
-    :::
+To display an archived docs version in a browser:
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:75]: docs > documentation

File: docs/README-EDITING.md [hunk 26]
```diff
@@ -66,77 +46,130 @@ sitemap:
...
+1. Start a [local Handsontable docs server](./README.md#getting-started-with-handsontable-docs).
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:76]: docs > documentation

File: docs/README-EDITING.md [hunk 27]
```diff
@@ -66,77 +46,130 @@ sitemap:
...
+// Wrong:
+/** {@link HiddenRows} */
 
-### Editing an archival version
+// OK:
+/** {@link hidden-rows HiddenRows} */
+```
 
-Directory: `/[number]/`.
+## Docs versioning
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:115]: docs > documentation

File: docs/README-EDITING.md [hunk 28]
```diff
@@ -66,77 +46,130 @@ sitemap:
...
 
-In a browser please go into: `/docs/[number]`.
+To create a new version of the Handsontable docs:
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:117]: docs > documentation

File: docs/README-EDITING.md [hunk 29]
```diff
@@ -66,77 +46,130 @@ sitemap:
...
-## Editing API Reference
+* From the `handsontable/docs` directory, run:
+```bash
+npm run docs:version <semver.version>
+# for example:
+# npm run docs:version 9.0
+```
 
-To edit API Reference goes into source code and change jsdocs comments. Then, to generate API Reference pages:
+To remove an existing version of the Handsontable docs:
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:126]: docs > documentation

File: docs/README-EDITING.md [hunk 30]
```diff
@@ -66,77 +46,130 @@ sitemap:
...
-npm run docs:api
+* Remove the required version's [directory](./README.md#handsontable-docs-directory-structure):
+```bash
+rm -rf ./<semver.version>
 ```
 
-## Versioning
+## Markdown containers
+
+To render content in different ways, the docs use custom Markdown containers, for example:
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:135]: docs > documentation

File: docs/README-EDITING.md [hunk 31]
```diff
@@ -66,77 +46,130 @@ sitemap:
...
-**Remove a version**
+| Container                    | Usage                                                      |
+|------------------------------|------------------------------------------------------------|
+| `::: tip`                    | A note.                                                    |
+| `::: example [options]`      | Renders a code example as specified in [options].          |
+| `::: source-code-link [URL]` | Adds a source code link to the right of an API ref header. |
 
-It is enough to remove version directory.
+### Adding code examples
+Using the `example` Markdown container, you can add code snippets that will show the code's result:
```
Comment (krzysztofspilka) [inline docs/README-EDITING.md:154]: Let's avoid future tense in documentation

File: docs/README.md [hunk 32]
```diff
@@ -1,60 +1,93 @@
...
-    helpers.js                  # Common helpers to get sidebars and list of versions
-    highlight.js                # Code highlight configuration
-```
+# Handsontable docs
+
+We treat documentation as an integral part of the Handsontable developer experience.
+
+That's why Handsontable comes with a vibrant, frequently-updated documentation portal.
+
+View the docs' latest production version at [handsontable.com/docs](https://handsontable.com/docs).
```
Comment (krzysztofspilka) [inline docs/README.md:7]: docs > documentation

File: docs/README.md [hunk 33]
```diff
@@ -1,60 +1,93 @@
...
+
+## About Handsontable docs
+
+The Handsontable docs are made up of three main sections:
```
Comment (krzysztofspilka) [inline docs/README.md:11]: docs > documentation

File: docs/README.md [hunk 34]
```diff
@@ -1,60 +1,93 @@
...
-```
+# Handsontable docs
+
+We treat documentation as an integral part of the Handsontable developer experience.
+
+That's why Handsontable comes with a vibrant, frequently-updated documentation portal.
+
+View the docs' latest production version at [handsontable.com/docs](https://handsontable.com/docs).
+
+## About Handsontable docs
```
Comment (krzysztofspilka) [inline docs/README.md:9]: docs > documentation

File: docs/README.md [hunk 35]
```diff
@@ -1,60 +1,93 @@
...
-    - /tools            
-        + /jsdoc-convert        # Tool used for converting jsdoc to markdown
-        + /version              # Tool used for rollup a new version
-        check-links.js          # Tool which check all links in the documentation
-        utils.js                # Utilities for tools
-    config.js                   # Configuration file
-    helpers.js                  # Common helpers to get sidebars and list of versions
-    highlight.js                # Code highlight configuration
-```
+# Handsontable docs
```
Comment (krzysztofspilka) [inline docs/README.md:1]: docs > documentation
Handsontable docs > Documentation overview? Currently, it seems to be too similar to the header below it.

File: docs/README.md [hunk 36]
```diff
@@ -1,60 +1,93 @@
...
+That's why Handsontable comes with a vibrant, frequently-updated documentation portal.
+
+View the docs' latest production version at [handsontable.com/docs](https://handsontable.com/docs).
+
+## About Handsontable docs
+
+The Handsontable docs are made up of three main sections:
+- **Guides**: Carefully-written implementation guides.
+- **Examples**: Code examples covering real-life use cases.
+- **API reference**: Full API reference, automatically generated from the source code of a required Handsontable version.
```
Comment (krzysztofspilka) [inline docs/README.md:14]: Full API reference > API reference
of a required Handsontable version > of each [Handsontable] version respectively (?)

File: docs/README.md [hunk 37]
```diff
@@ -1,60 +1,93 @@
...
+
+We update the docs:
```
Comment (krzysztofspilka) [inline docs/README.md:16]: docs > documentation

File: docs/README.md [hunk 38]
```diff
@@ -1,60 +1,93 @@
...
+- With every Handsontable version release.
+- Whenever there's something new to add to improve.
+
+**See also:**
+
+* [Docs editing guidelines](./README-EDITING.md)
+* [Docs deployment guidelines](./README-DEPLOYMENT.md)
```
Comment (krzysztofspilka) [inline docs/README.md:23]: docs > documentation

File: docs/README.md [hunk 39]
```diff
@@ -1,60 +1,93 @@
...
+- **Examples**: Code examples covering real-life use cases.
+- **API reference**: Full API reference, automatically generated from the source code of a required Handsontable version.
+
+We update the docs:
+- With every Handsontable version release.
+- Whenever there's something new to add to improve.
+
+**See also:**
+
+* [Docs editing guidelines](./README-EDITING.md)
```
Comment (krzysztofspilka) [inline docs/README.md:22]: docs > documentation

File: docs/README.md [hunk 40]
```diff
@@ -1,60 +1,93 @@
...
+
+1. From the `handsontable/docs` directory, install the docs dependencies:
+    ```bash
+    npm install
+    ```
+2. From the `handsontable/docs` directory, generate the API reference:
+   ```bash
+   npm run docs:api
+   ```   
+3. From the `handsontable/docs` directory, start your local docs server:
```
Comment (krzysztofspilka) [inline docs/README.md:37]: docs > documentation

File: docs/README.md [hunk 41]
```diff
@@ -1,60 +1,93 @@
...
+   ```bash
+   npm run docs:start
+   ```
+4. In your browser, go to: http://localhost:8080/docs/.
+
+## Handsontable docs npm scripts:
```
Comment (krzysztofspilka) [inline docs/README.md:43]: docs > documentation
Handsontable docs npm scripts: > Documentation npm scripts

**Review by krzysztofspilka — CHANGES_REQUESTED** [review]

File: docs/README-EDITING.md (hunk 15 above)
Comment (kirszenbaum) [reply docs/README-EDITING.md:37]: @krzysztofspilka it's a bit of a mistake on VuePress's part ("front matter" is much more common), but sure, let's keep it consistent with their docs

**Comment by kirszenbaum** [comment]
Thanks for the review @krzysztofspilka ! All changes applied. I'll note your stylistic preferences so I can use them when creating a style guide in the future

File: docs/README-EDITING.md [hunk 42]
```diff
@@ -66,77 +46,128 @@ sitemap:
...
-rm -rf ./[NUMBER]
+```js
+::: example #exampleId .class :preset --html 1 --js 2 --css 3  --no-edit --tab preview
+// `#example1` is the example's ID, for creating a Handsontable container
+
+    ```js
+    // code here
+    ```
+:::
 ```
```
Comment (Siemienik) [inline docs/README-EDITING.md:162]: Thank you for adding this example of how to insert code preview. I think it is fair to add two snippets more inside (HTML, CSS):

```js
::: example #exampleId .class :preset --html 1 --js 2 --css 3  --no-edit --tab preview
// `#example1` is the example's ID, for creating a Handsontable container
    ```html
    <div id="example1"></div>
    ```
    ```js
    // code here
    ```
    ```css
    /* custom css here */
    ```
:::
```

File: docs/README-EDITING.md [hunk 43]
```diff
@@ -66,77 +46,128 @@ sitemap:
...
+
+The `example` Markdown container offers the following options:
+
+| Option           | Required | Example           | Possible values                                            | Usage                          |
+|------------------|----------|-------------------|------------------------------------------------------------|--------------------------------|
+| `#exampleId`     | Yes      | `#data-grid-1`    | String                                                     | Container's unique ID.         |
+| `.class`         | No       | `.new-class`      | String                                                     | Container's custom class.      |
+| `:preset`        | No       | `:react`          | [`preset`](.vuepress/handsontable-manager/dependencies.js) | Sets code dependencies.        |
+| `--codeOption 1` | Yes      | `--html 1 --js 2` | [`codeOption`](.vuepress/highlight.js) + Number            | Sets code snippets' positions. |
```
Comment (Siemienik) [inline docs/README-EDITING.md:171]: Could you divide it to separated position?

| `--js <pos>` | No | `--js 1` | Positive Integer, default 1 | Sets the position of code snippet in a markdown container |
| `--html [pos]` | No | `--html 2` | Positive Integer, default 0 (0=disabled)  | Sets the position of html snippet in a markdown container. The 0 value disables the HTML tab |
| `--css [pos]` | No | `--css 2` | Positive Integer, default 0 (0=disabled)  | Sets the position of css snippet in a markdown container. The 0 value disables the CSS tab |

File: docs/README.md [hunk 44]
```diff
@@ -1,60 +1,93 @@
...
+
+## Getting started with Handsontable documentation
+
+To start a local Handsontable documentation server:
+
+1. From the `handsontable/docs` directory, install the documentation dependencies:
+    ```bash
+    npm install
+    ```
+2. From the `handsontable/docs` directory, generate the API reference:
```
Comment (Siemienik) [inline docs/README.md:33]: As the Readme is in the `docs` directory,  I don't think so, that repeating `From the `handsontable/docs` directory,` is required here.

File: docs/README.md [hunk 45]
```diff
@@ -1,60 +1,93 @@
...
+│   ├── handsontable-manager    # A module that runs Handsontable examples in different Handsontable versions
+│   ├── public                  # The documentation's public (static) assets
+│   ├── theme                   # Theme overwrites and customizations
+│   ├── tools                   # Our custom documentation tools
+│   │   ├── check-links.js      # The documentation's link checker
+│   │   ├── jsdoc-convert       # JSDoc-to-Markdown converter
+│   │   ├── utils.js            # Tools utilities
+│   │   └── version             # A tool that creates new documentation versions
+│   ├── config.js               # VuePress configuration
+│   ├── docs-links.js           # Lets us link to the documentation's latest version with `@` (e.g. [link](@/guides/path/file.md.)
```
Comment (Siemienik) [inline docs/README.md:79]: It does not link to the `latest`. it links in the scope of the currently selected version.

For instance:

* `[link](@/api/metaSchema.md#autorowsize)` in `/next/` generate link:`[link](/docs/next/api/options#autorowsize)`
* `[link](@/api/metaSchema.md#autorowsize)` in `/9.0/` (latest) generate link:`[link](/docs/api/options#autorowsize)`
* `[link](@/api/metaSchema.md#autorowsize)` in `/8.3/` generate link:`[link](/docs/8.3/api/options#autorowsize)`

File: docs/README.md [hunk 46]
```diff
@@ -1,60 +1,93 @@
...
+│   ├── handsontable-manager    # A module that runs Handsontable examples in different Handsontable versions
+│   ├── public                  # The documentation's public (static) assets
+│   ├── theme                   # Theme overwrites and customizations
+│   ├── tools                   # Our custom documentation tools
+│   │   ├── check-links.js      # The documentation's link checker
+│   │   ├── jsdoc-convert       # JSDoc-to-Markdown converter
+│   │   ├── utils.js            # Tools utilities
+│   │   └── version             # A tool that creates new documentation versions
+│   ├── config.js               # VuePress configuration
+│   ├── docs-links.js           # Lets us link within the currently-selected docs version with `@` (e.g. [link](@/guides/path/file.md.)
```
Comment (Siemienik) [inline docs/README.md:79]:
```suggestion
│   ├── docs-links.js           # Lets us link within the currently-selected docs version with `@` (e.g. [link](@/guides/path/file.md).)
```

**Review by Siemienik — APPROVED** [review]
Great done, Thank you 🥇

**Comment by krzysztofspilka** [comment]
LGTM

**Pull request merged**